### ISSUE TITLE
Please consider re-exporting Bytes

### ISSUE THREAD

**Issue by km19809** [igumnoff/shiva#131]
To implement the Transformer trait, or create an ImageData struct, we need Bytes.
If this crate re-export `bytes::Bytes`, it will be handy.

**Comment by evgenyigumnov** [comment]
```rust
#[derive(Debug, Clone, PartialEq, Default)]
#[cfg_attr(feature = "json", derive(Serialize, Deserialize))]
pub struct ImageData {
    #[cfg_attr(feature = "json", serde(skip))]
    bytes: Bytes,
    title: String,
    alt: String,
    image_type: ImageType,
    align: ImageAlignment,
    size: ImageDimension,
}
```

it is already use bytes::Bytes

What do you mean?

**Comment by km19809** [comment]
A rust crate can re-export their dependencies. [RustDoc](https://doc.rust-lang.org/rustdoc/write-documentation/re-exports.html)
These dependencies are often called "public dependencies". [#](https://rust-lang.github.io/api-guidelines/necessities.html#public-dependencies-of-a-stable-crate-are-stable-c-stable) 
Shiva relies on the `bytes` crate, forcing users to add the dependency.
Re-exporting can mitigate this issue.
[This article](https://www.lurklurk.org/effective-rust/re-export.html) explains the effectiveness of re-exporting.
(I am not an English native, so my words may be awkward; The article above will help!)

**Comment by evgenyigumnov** [comment]
fixed in 1.4.8
